ios/FluentUI.Demo/FluentUI.Demo/Demos/AvatarGroupDemoController.swift

```diff
@@ -0,0 +1,289 @@
+        addRow(text: "Overflow count", items: [UIStackView(arrangedSubviews: [overflowCountTextField, overflowCountButton.view])], textStyle: .footnote, textWidth: Constants.settingsTextWidth)
+
+        addTitle(text: "Avatar Stack No Border")
+        insertAvatarViews(style: .stack, showBorders: false)
+        addTitle(text: "Avatar Stack With Border")
+        insertAvatarViews(style: .stack, showBorders: true)
+        addTitle(text: "Avatar Stack With Mixed Border")
+        insertAvatarViews(style: .stack, showBorders: true, mixed: true)
+        addTitle(text: "Avatar Pile No Border")
+        insertAvatarViews(style: .pile, showBorders: false)
+        addTitle(text: "Avatar Pile With Border")
+        insertAvatarViews(style: .pile, showBorders: true)
+        addTitle(text: "Avatar Pile With Mixed Border")
+        insertAvatarViews(style: .pile, showBorders: true, mixed: true)
+    }
+
+    private struct Constants {
+        static let settingsTextWidth: CGFloat = 200
+        static let avatarsTextWidth: CGFloat = 100
+        static let maxTextInputCharCount: Int = 4
+    }
+
+    private var avatarGroups: [MSFAvatarGroup] = []
+
+    private var avatarCount: Int = 5 {
+        didSet {
+            reloadAvatarViews()
+        }
+    }
+
+    private var isUsingAlternateBackgroundColor: Bool = false {
+        didSet {
+            updateBackgroundColor()
+        }
+    }
+
+    private func updateBackgroundColor() {
+        view.backgroundColor = isUsingAlternateBackgroundColor ? UIColor(light: Colors.gray100, dark: Colors.gray600) : Colors.surfacePrimary
+    }
+
+    @objc private func toggleAlternateBackground(switchView: UISwitch) {
+        isUsingAlternateBackgroundColor = switchView.isOn
+    }
+
+    private var maxDisplayedAvatars: Int = 4 {
+        didSet {
+            if oldValue != maxDisplayedAvatars {
+                maxAvatarsTextField.text = "\(maxDisplayedAvatars)"
+
+                for avatarGroup in avatarGroups {
+                    avatarGroup.state.maxDisplayedAvatars = maxDisplayedAvatars
+                }
+            }
+        }
+    }
+
+    private var overflowCount: Int = 0 {
+        didSet {
+            if oldValue != overflowCount {
+                overflowCountTextField.text = "\(overflowCount)"
+
+                for avatarGroup in avatarGroups {
+                    avatarGroup.state.overflowCount = overflowCount
+                }
+            }
+        }
+    }
+
+    private func insertAvatarViews(style: MSFAvatarGroupStyle, showBorders: Bool, mixed: Bool = false) {
+        var constraints: [NSLayoutConstraint] = []
+
+        for size in MSFAvatarSize.allCases.reversed() {
+            let containerView = UIView(frame: .zero)
+
+            let avatarGroup = MSFAvatarGroup(style: style, size: size)
+            var border = showBorders
+            for index in 0...avatarCount - 1 {
+                vSamplePersonas[index].isRingVisible = border
```

Comment:
nit: we don't usually have "v" to convey vectors in variable names. I think you can call this just samplePersonas.

Reply:
There is already a samplePersonas array (line 248) that contains all the persona data. I added the v to indicate it was used for vNext

Reply:
Oh! I didn't catch the idea that "v" indicates vNext.
How about changing the name to **samplePersonaStates** or something that is more descriptive like that?